client/src/main/scala/org/apache/celeborn/client/CommitManager.scala

```diff
@@ -250,6 +250,10 @@ class CommitManager(appUniqueId: String, val conf: CelebornConf, lifecycleManage
     getCommitHandler(shuffleId).isStageEnd(shuffleId)
   }
 
+  def isStageEndOrInProcess(shuffleId: Int): Boolean = {
```

Comment:
nit: IMO it's unnecessary to add this method.

Reply:
i just follow `isStageEnd` usage in `CommitManager`